File: src/codes/Users_Manual/sender.py
~from pyautd3 import AUTD3, Controller, Duration, Null, SenderOption, FixedSchedule, ParallelMode
~from pyautd3.link.nop import Nop
~autd = Controller.open([AUTD3()], Nop())
sender = autd.sender(
    SenderOption(
        send_interval=Duration.from_millis(1),
        receive_interval=Duration.from_millis(1),
        timeout=None,
        parallel=ParallelMode.Auto,
        strict=True
    ),
    FixedSchedule(),
)
~d = Null()
sender.send(d)

File: src/codes/Users_Manual/sender.rs
# use autd3::prelude::*;
# use std::time::Duration;
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::open([AUTD3::default()], autd3::link::Nop::new())?;
let mut sender = autd.sender(
    SenderOption {
        send_interval: Duration::from_millis(1),
        receive_interval: Duration::from_millis(1),
        timeout: None,
        parallel: ParallelMode::Auto,
        strict: true
    },
    FixedSchedule::default(),
);
# let d = Null {};
sender.send(d)?;
# Ok(())
# }